pkg/controller/migstorage/validation.go

```diff
@@ -123,7 +112,8 @@ func (r ReconcileMigStorage) validateBackupStorage(storage *migapi.MigStorage) e
 			Status:   True,
 			Reason:   NotFound,
 			Category: Critical,
-			Message:  InvalidBSCredsSecretRefMessage,
+			Message: fmt.Sprintf("The `backupStorageConfig.credsSecretRef` must reference a valid `secret`, context %s",
```

Comment:
I know I recommended something like _content_ but after seeing it, I wonder if `key` would be better?

Reply:
yeah I like instead of `key` instead of `content` or `context`, +1